src/helpers/utils.js

```diff
@@ -3,12 +3,14 @@ import semverCompare from "semver-compare";
 
 export const versionFilter = (name, prevVersion, nextVersion) => {
   const tokenizedNextVersion = nextVersion.split(".");
-  return prevVersion
+  const afterPrevVersion = prevVersion
     ? semverCompare(name, prevVersion) >= 0
     : name.startsWith(
         prevVersion ||
           [tokenizedNextVersion[0], tokenizedNextVersion[1]].join(".")
       );
+  const beforeNextVersion = semverCompare(name, nextVersion) <= 0;
+  return afterPrevVersion && beforeNextVersion;
```

Comment:
What does this do?
Could you add a test to show what it's solving?

Reply:
It's to make sure if you set `--next-version` to a release that's not the latest release it doesn't include any later releases in the output. It was useful when I was developing, I can see if I can add a test

Reply:
Or I can remove it, like I said it was to make it easier to test during dev, we don't need it for anything in our release process

Reply:
Added test b279ab656a13f8c9c62c0aefe9bc41d4d46cc942